src/Microsoft.Data.Analysis.nuget/Microsoft.Data.Analysis.nuget.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
The aim here is to start including MDA.Interactive.dll into MDA.Nuget right? I think the existing MDA.csproj is already creating MDA.Nuget, so we don't need this new csproj? Instead we may need to add the add the properties that @eerhardt suggests, but in such a way to result in MDA.Interactive.dll landing in a place that the notebook can pick up the formatter automatically. Give me a few minutes while I experiment locally.

Reply:
Alright, so I think the right thing to do here is wait for @jonsequitur to chime in. Specifically, does an extension's Nuget package need to have a specific structure for the notebook to pick it up? 

Adding a new csproj here to package both MDA and the formatter is a valid approach. It's just that we now have to point notebooks users to the new nuget package instead of MDA.nupkg right? Ideally, the formatter would ship with MDA.nupkg. That means the formatting code should live within MDA.csproj (not something I'm in favor of at the moment) or we have to use advanced dotnet pack rules: https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets#advanced-extension-points-to-create-customized-package?

Reply:
Sorry I missed this comment. We have samples for building extensions here: https://github.com/dotnet/interactive/tree/master/samples/extensions. The approach we'll want to take here is to create a separate project that references both Microsoft.Data.Analysis and Microsoft.DotNet.Interactive. The two projects' outputs get packed into a single NuGet package using a third project like this: https://github.com/dotnet/interactive/blob/master/samples/extensions/Library.nuget/Library.nuget.csproj.

@dcostea, I'm happy to help out here. 

We'll also want to update to the latest Microsoft.DotNet.Interactive packages as we've made some minor breaking changes.

Reply:
Jon, thank you!

Instead of adding
```<ItemGroup>
   <None Include="../Library/bin/$(Configuration)/$(TargetFramework)/*.dll"  Pack="true" PackagePath="lib/$(TargetFramework)" />
  </ItemGroup>
```
I have used ```<IsPackable>false</IsPackable>``` in Microsoft.Data.Analysis.Interactive and Microsoft.Data.Analysis.Interactive csproj files, but I'm not sure this is acceptable.

The dependency on Microsoft.DotNet.Interactive is the latest.

@jonsequitur, could you review the latest commit?